inst/include/interface/rcpp/rcpp_objects/rcpp_fleet.hpp

```diff
@@ -61,20 +63,25 @@ class FleetInterface : public FleetInterfaceBase {
   std::string name = "NA"; /**< the name of the fleet */
   bool is_survey = false; /**< whether this is a survey fleet */
   int nages;              /**< number of ages in the fleet data*/
+  int nlengths = 0;       /**< number of length bins in the fleet data */
   int nyears;             /**< number of years in the fleet data */
   ParameterVector log_q;  /**< log of catchability for the fleet*/
   ParameterVector
       log_Fmort;           /**< log of fishing mortality rate for the fleet*/
   ParameterVector log_expected_index; /**< expected index of abundance for the survey */
   ParameterVector proportion_catch_numbers_at_age; /**< expected catch numbers at age for the fleet */
+  ParameterVector proportion_catch_numbers_at_length; /**< expected catch numbers at age for the fleet */
+  ParameterVector age_length_conversion_matrix; /**< matrix to convert ages to lengths */
```

Comment:
@nathanvaughan-NOAA should this have a default value in case there are no length data and a conversion matrix is not needed? I am wondering because right now `m_length_at_age()` returns nothing if the data are not present, which I think is causing some errors.

Reply:
Thanks @kellijohnson-NOAA I thought that these should be safe because the length code only runs when nlengths is >0 but I'll take a look and double check and figure out where it's best to add a catch for this.